core/src/main/resources/fabric8/default-kind-filename-mapping.adoc

```diff
@@ -0,0 +1,138 @@
+|===
+|Kind |Filename
```

Comment:
The first column is ist the filename part indicating the kind, the second then is the kind.

Maybe `Filename` is also a bit misleading as one could think about this as a whole file name but its only the last part before the extension. Hard to find a good name here, as the extension is obviously ".yml", so it something like a 'file indicator' or so. wdyt ?

Reply:
Tbh, I wouldn't change the header but the order of the columns because I expect users to look up which file indicator is used for which Kind, so look up the Kind (e.g. 'ConfigMap') to find out what to use in the fragment file (`-cm.yml`). As the table is supposed to be ordered alphabetically in the first column, it would be awesome if the kind would be in the first column.

I know this required some code and test changes, but I believe it would be worth before go out in the wild with it.

Reply:
In the original solution, https://github.com/fabric8io/fabric8-maven-plugin/blob/5e52eded75357bc8df3ac4ce93e931bd39c75e23/core/src/main/java/io/fabric8/maven/core/util/kubernetes/KubernetesResourceUtil.java#L198-L244 first was the "short" version and second the "long" version so what you suggest here is to leave the header and change the columns, ok, it should not be hard at all.

About the name, how about `filename-suffix`?

Reply:
> About the name, how about `filename-suffix`?

Sound goods. We could also call it 'filename type' (not sure if 'suffix' pronounces the location of this type a bit too much). But never mind, we can change that easily later when we found something better.